tsconfig.json

```diff
@@ -19,12 +19,12 @@
 		"noUnusedParameters": false,
 		"noUnusedLocals": false,
 
-		"sourceMap": true
+		"sourceMap": true,
 
-	},
-
-	"files": [
```

Comment:
As long as you have the tsconfig in the root of the project/module I've had no problems with it finding modules.